diff --git a/pumpkin-crates/core/src/lib.rs b/pumpkin-crates/core/src/lib.rs
index 644183a8..7867a05f 100644
--- a/pumpkin-crates/core/src/lib.rs
+++ b/pumpkin-crates/core/src/lib.rs
@@ -297,6 +297,8 @@
 //!     }
 //! }
 //!  ```
+#![feature(unsigned_is_multiple_of)]
+
 #[cfg(doc)]
 use crate::results::unsatisfiable::UnsatisfiableUnderAssumptions;
 pub(crate) mod basic_types;
@@ -335,3 +337,5 @@ pub use crate::api::solver::DefaultBrancher;
 pub use crate::api::solver::Solver;
 pub use crate::basic_types::ConstraintOperationError;
 pub use crate::basic_types::Random;
+
+
diff --git a/pumpkin-solver/build.rs b/pumpkin-solver/build.rs
index 86b897b5..6a18d51a 100644
--- a/pumpkin-solver/build.rs
+++ b/pumpkin-solver/build.rs
@@ -10,19 +10,7 @@ fn main() {
 }
 
 fn run() -> Result<(), Box<dyn Error>> {
-    compile_c_binary(&["tests/cnf/checkers/drat-trim.c"], "drat-trim")?;
-    compile_c_binary(&["tests/cnf/checkers/precochk.c"], "precochk")?;
-    compile_c_binary(
-        &[
-            "tests/wcnf/checkers/maxsat-checker.cc",
-            "tests/wcnf/checkers/Wcnf.cc",
-        ],
-        "maxsat-checker",
-    )?;
-
-    println!("cargo:rerun-if-changed=tests/cnf/checkers/");
-    println!("cargo:rerun-if-changed=tests/wcnf/checkers/");
-
+    
     println!("cargo:rerun-if-changed=build.rs");
 
     Ok(())
